docs/rules/comma-style.md

```diff
@@ -1,8 +1,8 @@
 # Comma style (comma-style)
 
-Comma Style rule enforces comma styles for a list of things separated by commas. There are two comma styles primarily in JavaScript. The standard one in which commas are placed at the end of the line. And Comma-First, in which, commas are placed at the start of the next line after the list item.
+The Comma Style rule enforces styles for comma-separated lists. There are two comma styles primarily used in JavaScript. The standard one in which commas are placed at the end of the line, and Comma First, in which commas are placed at the start of the next line after the list item.
```

Comment:
Would it help or hurt to reorganize the information in the third sentence as two items in a bulleted list?